PROVISIONING.md

```diff
@@ -0,0 +1,72 @@
+# Discovery and Provisioning
+
+## Docs
+- please look at https://github.com/silicom-ltd/uBMC
+- please look at https://opencomputeproject.github.io/onie/overview/
+- please look at https://fidoalliance.org/intro-to-fido-device-onboard/
+- please look at https://www.rfc-editor.org/rfc/pdfrfc/rfc8572.txt.pdf (Secure Zero Touch Provisioning (SZTP))
+- please look at https://pypi.org/project/sztpd/
+- please look at https://watsen.net/docs/sztpd/0.0.11/admin-guide/
+
+## Re-Provisioning
+
+tbd... re-celling, faults...
+
+## Manual Provisioning
+
+tbd...
+
+## USB/Virtual media Provisioning
```

Comment:
Just an observation: seems to me that we have two main use cases for provisioning - large scale deployments (where automation and security are major drivers), and small scale, unique, specialized deployments, where this might be a desirable pattern. most of the discussion seems to be focused on the former, not so much on the latter. Maybe we should be more explicit on the motivation for the various "paths" for provisioning.

Reply:
@shachartal I agree I didn't have chance to fill details on `small scale, unique, specialized deployments` can I ask you to add more details and raise a PR here please ?